api/init/security/oauth/github.py

```diff
@@ -0,0 +1,71 @@
+import json
+import os
+from flask import redirect, request, session
+from flask_restplus import Namespace, Resource
+from requests_oauthlib import OAuth2Session
+from security.token import get_jwt_token, TokenType, get_token_redirect_response
+
+# pylint: disable=unused-variable
+
+# OAuth endpoints given in the GitHub API documentation
+AUTHORIZATION_URI = 'https://github.com/login/oauth/authorize'
+TOKEN_URI = 'https://github.com/login/oauth/access_token'
+USER_PROFILE_URI = 'https://api.github.com/user'
+USER_EMAIL_URI = 'https://api.github.com/user/emails'
+SCOPE = ['user:email']
+
+# OAuth application configuration created on Github
+client_id = os.environ['GITHUB_CLIENT_ID']
+client_secret = os.environ['GITHUB_CLIENT_SECRET']
+redirect_uri = os.environ['GITHUB_REDIRECT_URI']
+
+
+def get_user_info(github_session: object):
+    """Gets user profile using OAuth session."""
+
+    user_profile = github_session.get(USER_PROFILE_URI).content.decode('utf-8')
+    user_profile = json.loads(user_profile)
+
+    if user_profile['email'] is None:
```

Comment:
Are you sure, you don't mean something like this?
```suggestion
    if 'email' in user_profile:
```